spec/factories/tema.rb

```diff
@@ -23,7 +23,7 @@
 
     factory :tema_titular do
       fecha_titulares { 10.minutes.ago }
-      descripcion     { Faker::Lorem.sentences.join("/r/n") }
+      descripcion     { FFaker::Lorem.sentences.join("/r/n") }
```

Comment:
Unnecessary spacing detected.<br>Prefer single-quoted strings when you don't need string interpolation or special symbols.